composer/datasets/in_context_learning_evaluation.py

```diff
@@ -163,10 +169,10 @@ def __init__(
         self.max_answer_length = 0
         fewshot_rng = random.Random(fewshot_random_seed)
         self.encoded_dataset = self.prep_examples(num_fewshot, prompt_string, example_delimiter, continuation_delimiter,
-                                                  question_prelimiter, fewshot_rng)
+                                                  question_prelimiter, cot_delimiter, fewshot_rng)
 
     def prep_examples(self, num_fewshot: int, prompt_string: str, example_delimiter: str, continuation_delimiter: str,
-                      question_prelimiter: str, fewshot_rng: random.Random):
+                      question_prelimiter: str, cot_delimiter: str, fewshot_rng: random.Random):
```

Comment:
Does it make sense to default `cot_delimiter: str = ''`?

Reply:
Or `cot_delimiter: Optional[str] = None`? Will there be a case where some people do CoT without a specific delimiter (they use '') and some people who don't do CoT, in which case the delimiter should be `None` to represent the difference?

Reply:
yes I think this will be a common case!

Reply:
This argument should have a default value though.